src/main/java/com/github/onsdigital/babbage/api/endpoint/content/Resource.java

```diff
@@ -23,13 +36,72 @@
     @GET
     public void get(@Context HttpServletRequest request, @Context HttpServletResponse response) throws IOException {
         try {
-            ContentResponse contentResponse = ContentClient.getInstance().getResource(request.getParameter("uri"));
+            String uri = request.getParameter("uri");
+            String width = request.getParameter("width");
+
+            ContentResponse contentResponse = null;
+
+            try {
+                contentResponse = ContentClient.getInstance().getResource(uri);
+            } catch (ContentReadException e) {
+                System.out.println("Get resource returned error response, status: " + e.getStatusCode());
+                ErrorHandler.renderErrorPage(404, response);
+                return;
```

Comment:
I believe you can replace the above 3 lines with 
```
throw new ResourceNotFoundException(e.getMessage());
```
This will cause the equivalent of what you are doing - logging, populating the response, and 'returning' from the function.